public/js/dashboardInfoForm.js

```diff
@@ -1,43 +1,46 @@
 /* global google*/
 
 /**
- * @param {string} id Organization's unique database ID
- * @return {JSON} An organization's information
+ * Gets the information of the organization assigned to the current user.
+ * @return {JSON} An organization's information.
  */
-function getFormInfo(id) {
-  // TODO: Get organization's information from database
-  const info = {
-    ID: 'google1',
-    name: 'Google',
-    placesID: '12345 Road Drive',
-    description: 'a great place to find non profits!',
-    phone: '1234567890',
-    email: 'org@google.com',
-    websiteURL: 'google.com',
-  };
-  return info;
+async function getFormInfo() {
+  const memberData = await (await fetch('/data/member')).json();
+  const organizationData = await (await fetch(`/data/organization/member/${memberData.id}`)).json();
+  const orgInfo = await (await fetch(`/data/organizations/${organizationData.id}`)).json();
```

Comment:
Does `(await fetch()).json` work for these lines instead of using two awaits?

Reply:
Unfortunately it doesn't: https://gist.github.com/msmfsd/fca50ab095b795eb39739e8c4357a808#gistcomment-2085059.